model/src/main/scala/lucuma/schemas/enums/ProposalAttachmentType.scala

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.schemas.enums
+
+import cats.Eq
+import cats.derived.*
+import io.circe.*
+import io.circe.generic.semiauto
+import lucuma.core.util.Display
+import lucuma.core.util.Enumerated
+import monocle.Focus
+import monocle.Lens
+
+case class ProposalAttachmentType(
+  tag:       String,
+  shortName: String,
+  longName:  String
+) derives Eq:
+  val accept: String = ".pdf" // only PDF files are currently allowed
```

Comment:
Can we have this as a constant on the object?

Reply:
Yeah. We can. I was just anticipating the day when `they` decide that we need to support different file types for different attachment types, like we do for obs attachments. But, that's probably a bit too paranoid and easy to fix, anyway.